lib/resque.rb

```diff
@@ -165,6 +165,12 @@ def prune_interval
     @prune_interval || DEFAULT_PRUNE_INTERVAL
   end
 
+  attr_writer :enqueue_front
+  def enqueue_front
+    return @enqueue_front unless @enqueue_front.nil?
+    @enqueue_front = false
+  end
```

Comment:
Ah yes, I see. It's the same thing in the sense that where this is _used_, it has the same effect, not that it's the same at this line of code. :+1: